shanoir-ng-front/src/app/async-tasks/status/task-status.component.html

```diff
@@ -45,7 +45,11 @@
             <li *ngIf="task.route">
                 <label>Link</label>
                 <span class="right-col left-icon">
-                    <a [routerLink]="task.route">
+                    <a *ngIf="task._eventType != 'downloadStatistics.event'" [routerLink]="task.route">
+                        <i class="fa-solid fa-link"></i>
+                        {{task.route}}
+                    </a>
+                    <a *ngIf="task._eventType == 'downloadStatistics.event'" (click)="downloadStats($event)" href="#">
```

Comment:
When I click on the event in the job list, I have an error, i need to click on the "link" to be able to download it